src/views/statutory/VotesAmounts.vue

```diff
@@ -6,10 +6,12 @@
 
         <div class="subtitle">By antenna</div>
 
+        <div> (ofc, add an icon here, and hide this block as long as the data is not loaded) <a :href="exportVotesCSV.content" :download="exportVotesCSV.filename">download list of votes</a> </div>
```

Comment:
this is even smarter than I did, kudos for that, I have no idea that you can put the content of csv as href